cmd/swarmctl/network/common.go

```diff
@@ -32,6 +32,12 @@ func GetNetwork(ctx context.Context, c api.ControlClient, input string) (*api.Ne
 		}
 
 		if l := len(rl.Networks); l > 1 {
+			for _, n := range rl.Networks {
+				if n.Spec.Annotations.Name == input {
+					// Found a full name match
```

Comment:
It's a little surprise for me to know that the name could be non-unique. But it should not be a problem since the ID is still unique.